src/DepartmentService.Business/Department/GetDepartmentCommand.cs

```diff
@@ -1,304 +1,382 @@
+            IGetProjectsRequest.CreateObj(projectsIds));
+
+        if (response.Message.IsSuccess)
+        {
+          return response.Message.Body.Projects;
+        }
+
+        _logger.LogWarning(loggerMessage + "Reasons: {Errors}", string.Join("\n", response.Message.Errors));
+      }
+      catch (Exception exc)
+      {
+        _logger.LogError(exc, loggerMessage);
+      }
+
+      errors.Add("Can not get projects data. Please try again later.");
+
+      return new();
+    }
+
+    private async Task<List<ImageData>> GetUserImagesAsync(List<Guid> usersIds, List<string> errors)
+    {
+      if (usersIds is null || !usersIds.Any())
+      {
+        return null;
+      }
+
+      try
+      {
+        Response<IOperationResult<IGetImagesResponse>> response =
+          await _rcImages.GetResponse<IOperationResult<IGetImagesResponse>>(
+            IGetImagesRequest.CreateObj(usersIds, ImageSource.User));
+
+        if (response.Message.IsSuccess && response.Message.Body.ImagesData != null)
+        {
+          return response.Message.Body.ImagesData;
+        }
+
+        _logger.LogWarning(
+          "Errors while getting images by users ids: {UsersIds}.\nErrors: {Errors}",
+          string.Join(", ", usersIds),
+          string.Join('\n', response.Message.Errors));
+      }
+      catch (Exception exc)
+      {
+        _logger.LogError(
+          exc,
+          "Errors while getting images by users ids: {UsersIds}.",
+          string.Join(", ", usersIds));
+      }
+
+      errors.Add("Can not get images. Please try again later.");
+
+      return null;
+    }
+
+    private async Task<List<PositionData>> GetPositionsAsync(List<Guid> usersIds, List<string> errors)
+    {
+      if (usersIds is null || !usersIds.Any())
+      {
+        return null;
+      }
+
+      try
+      {
+        Response<IOperationResult<IGetPositionsResponse>> response =
+          await _rcGetPositions.GetResponse<IOperationResult<IGetPositionsResponse>>(
+            IGetPositionsRequest.CreateObj(
+            usersIds));
+
+        if (response.Message.IsSuccess)
+        {
+          return response.Message.Body.Positions;
+        }
+
+        _logger.LogWarning(
+          "Errors while getting positions of users ids {UserId}.\n Errors: {Errors}",
+          string.Join(", ", usersIds),
+          string.Join('\n', response.Message.Errors));
+      }
+      catch (Exception exc)
+      {
+        _logger.LogError(
+          exc,
+          "Can not get positions of users ids {UserId}.",
+          string.Join(", ", usersIds));
+      }
+
+      errors.Add("Can not get users positions. Please try again later.");
+
+      return null;
+    }
+
+    private async Task<List<NewsData>> GetNewsDataThroughBrokerAsync(List<Guid> newsIds, List<string> errors)
+    {
+      if(newsIds is null || !newsIds.Any())
+      {
+        return new();
+      }
+
+      string loggerMessage = $"Can not fet news data for specific news ids '{newsIds}'.";

```

Comment:
remove to 262 string, you use the variable only in 262 lane, no need create it here